src/test/java/de/cyface/collector/verticle/MainVerticleTest.java

```diff
@@ -100,7 +100,15 @@ private JsonObject config() throws IOException {
                 .put("http.endpoint", "/api/v3/")
                 .put("http.port", Network.freeServerPort(Network.getLocalHost()))
                 .put("salt", "abcdefg")
-                .put("mongo.db", mongoTest.clientConfiguration());
+                .put("mongo.db", mongoTest.clientConfiguration())
+                .put("admin.user", "admin")
+                .put("admin.password", "secret")
+                .put("jwt.expiration", 60)
+                .put("upload.expiration", 60)
```

Comment:
`jwt.expiration  = 60` is correct (seconds)

but `upload.expiration = 60` should probably be 60_000 (ms)